asamisellah/Maintenance-Tracker
# #157930395 Create tests for API endpoints

#### What does this PR do?
Create tests for getting, updating and creating requests

#### Description of tasks to be completed?
- Write test for request creation
- Write test for getting all user requests
- Write test for updating a request
- Write test for deleting a request
- Write test for signing up user
- Write test for signing in user

#### How should this be manually tested?
1. clone the [repo](https://github.com/asamisellah/Maintenance-Tracker)
2. open the folder in your text editor or IDE
3. `git chechout -b new-branch`
4. `git fetch ch-env-setup-157930685`

*Setup*
Make sure virtual environment is installed. Open terminal and run the following commands
1. `cd maintenance-tracker`
2.  `virtualenv venv` to create virtual environment
3.  `source venv/bin/activate` to run environment
4. `pip install -r requirements.txt`

*Testing*
To run tests for users `python test_user.py` 
To run for requests`python test_requests.py`

#### Any background context you want to provide?
None

#### What are the relevant pivotal tracker stories?
[#157930395 ](https://www.pivotaltracker.com/story/show/157930395/comments/190063220)

### Questions
Are the tests failing successfully?
